app/src/main/java/com/gh4a/utils/StringUtils.java

```diff
@@ -98,18 +101,18 @@ public static String formatName(String userLogin, String name) {
         return userLogin + (!StringUtils.isBlank(name) ? " - " + name : "");
     }
 
-    public static CharSequence formatRelativeTime(Context context, Date date, boolean showDateIfLongAgo) {
+    public static CharSequence formatRelativeTime(Context context, Date date, boolean showDateWithPreposition) {
```

Comment:
This redefinition of the parameter doesn't make sense to me. The meaning of the parameter is 'show timestamp if long ago' vs. 'show always relative times', e.g. '1 year ago' vs. 'on 2020/09/01'. How is that the same as '2020/09/01' vs. 'on 2020/09/01'?